index.ts

```diff
@@ -1,12 +1,17 @@
 import { V2Client, V2PromiseClient } from './proto/clarifai/api/service_grpc_web_pb'
 
 export class ClarifaiStub {
-  static grpc (hostname = 'api.clarifai.com') {
+  static grpc (hostname = 'https://api-grpc-web.clarifai.com') {
     const options = { 'grpc.max_receive_message_length': 128 * 1024 * 1024 } // 128 MB
     return new V2Client(hostname, null, options)
   }
+
+  static promise (hostname = 'https://api-grpc-web.clarifai.com') {
```

Comment:
This looks like GRPC_WEB_HOST from env vars.

Reply:
It is, this is our production grpc-web hostname which is the intended hostname for external consumers of this package.